android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/EmailSignInRedirectUriActivity.java

```diff
@@ -0,0 +1,25 @@
+package org.sagebionetworks.bridge.android.manager;
+
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+import org.sagebionetworks.bridge.android.AuthManagementActivity;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class EmailSignInRedirectUriActivity extends AppCompatActivity {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        Intent launchIntent = new Intent(this, AuthManagementActivity.class)
+                .setData(getIntent().getData())
+        .addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_SINGLE_TOP);
```

Comment:
Was this pass-through activity added for the sole purpose of adding Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_SINGLE_TOP?

Reply:
Good point, this actually isn't needed because we're not using a Chrome Tab at any point. I copied this from CRF's AppAuth

Reply:
Ohhhh I ran into that too when doing fit bit auth, it was really annoying. The Chrome tab kept sticking around in all sorts of odd scenarios.  Thanks for the background info